jphyloref/lib/download_factpp_native_lib.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# This is a helper script to help Travis CI download the native libraries for
+# the FaCT++ reasoner. It is not intended to be used directly by users.
+
+cd /tmp
+wget "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/factplusplus/FaCTpp-linux-v1.5.2.tgz"
+tar zxvf FaCTpp-linux-v1.5.2.tgz
+cd -
+mv /tmp/FaCT++-linux-v1.5.2/64bit/libFaCTPlusPlusJNI.so jphyloref/lib
```

Comment:
Thanks for the change! The value was only that (1) there were several steps to be executed, so a script seemed neater, and (2) so that users could have a script to execute as well. However, they can always copy the one-liner from `.travis.yml`, so this looks good to me!

Sorry for not responding to your suggestion -- I totally missed it when I went through this PR on Monday!